spark-bigquery-connector-common/src/test/java/com/google/cloud/spark/bigquery/integration/WriteIntegrationTestBase.java

```diff
@@ -921,6 +923,45 @@ public void testWriteToCmekManagedTable() throws Exception {
         .isEqualTo(destinationTableKmsKeyName);
   }
 
+  @Test
+  public void testWriteNumericsToWiderFields() throws Exception {
```

Comment:
can we add an exception case as well?

Reply:
Why do we need it in the integration test, if the check is done locally?